rosdep/python.yaml

```
@@ -1129,6 +1129,10 @@ python-cookiecutter-pip:
   ubuntu:
     pip:
       packages: [cookiecutter]
+python-copier-pip:
```

The current guidance in [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1) states that pip keys should use the `python3-` prefix.

```suggestion
python3-copier-pip:
```

Reminder that you'll need to move the stanza after renaming the key to maintain alphabetical ordering.